test/integration/ynEIGEN/AssetRegistry.t.sol

```diff
@@ -141,7 +141,8 @@ contract AssetRegistryTest is ynEigenIntegrationBaseTest {
         assertEq(convertedAmount, expectedConvertedAmount, "Converted amount should match expected value based on real rate");
     }
 
-    function testsfrxETHConvertToUnitOfAccountFuzz(uint256 amount) public {
+    function testsfrxETHConvertToUnitOfAccountFuzz(uint256 amount) public skipOnHolesky {
+        vm.skip(block.chainid == 17000, "Impossible to test on Holesky");
```

Comment:
this already has modifier so instruction here i believe

Reply:
ups, I forgot to remove it after refactoring to the modifier